src/components/Calculator.js

```diff
@@ -0,0 +1,12 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+// import { Button } from './Button';
```

Comment:
- [ ] Kindly remove commented code for a cleaner codebase and more readable code

Reply:
Hi @Munsa1 !
Thanks for your feedback, you're right. I totally forgot to delete that comment.